docs/book/content/calibration/macro.md

```diff
@@ -5,48 +5,48 @@
+As discussed in the [OG-Core household theory documentation](https://pslmodels.github.io/OG-Core/content/theory/households.html), we use the elliptical disutility of labor function developed by {cite}`EvansPhillips:2017`.  We then fit the parameters of the elliptical utility function to match the marginal disutility from a constant Frisch elasticity function.  `OG-ZAF` users enter the constant Frisch elasticity as a parameter.  {cite}`Peterman:2016` finds a range of Frisch elasticities estimated from microeconomic and macroeconomic data.  These range from 0 to 4.  Peterman makes the case that in lifecycle models without an extensive margin for employment the  Frisch elasticity should be higher. For `OG-ZAF` we take a default value of 0.4 from {cite}`Altonji:1986`.
 
 ### Intertemporal elasticity of substitution
 
 The default value for the intertemporal elasticity of substitution, $\sigma$, is taken from {cite}`ABMW:1999`.  We set $\sigma=1.5$.
 
 ### Rate of time preference
 
-We take our default value for the rate of time preference parameter, $\beta$ from {cite}`Carroll:2009`.  We set the value of to $\beta=0.96$ (on an annual basis).
+We take our default value for the rate of time preference parameter, $\beta$ from {cite}`Carroll:2009`.  We set the value to $\beta=0.96$ (on an annual basis).
 
 
 ## Economic Assumptions
 
-As the default rate of labor augmenting technological change, $g_y$, we use a value of 3%.  The average annual growth rate in GDP per capita in the United States since 1948 is 2% per year.
+As the default rate of labor augmenting technological change, $g_y$, we use a value of 3%.  The average annual growth rate in GDP per capita in South Africa between 1961 and 2021 is 0.7% per year.
 
 ## Aggregate Production Function and Capital Accumulation
 
 The [OG-Core firm theory documentation](https://pslmodels.github.io/OG-Core/content/theory/firms.html) outlines the constant returns to scale, constant elasticity of substitution production function of the representative firm.  This function has two parameters; the elasticity of substitution and capital's share of output.
 
 ### Elasticity of substitution
 
-`OG-USA`'s default parameterization has an elasticity of substitution of $\varepsilon=1.0$, which implies a Cobb-Douglas production function.
+`OG-ZAF`'s default parameterization has an elasticity of substitution of $\varepsilon=1.0$, which implies a Cobb-Douglas production function.
 
 ### Capital's share of output
 
-The historical value in U.S. is about 0.33, but {cite}`EHS:2013` find capital's share is increasing.  We therefore use the slightly higher value of $\gamma =0.35$.  Note that the mean of capital's share of income from 1950 onwards is 0.38 from the Penn World Tables.
+Here, we use a value of $\gamma =1.0$ from the UN Data Portal API (see https://data.un.org).
```

Comment:
@NHgit213. This can't be right, gamma=1.0. The default parameters file [ogzaf_default_parameters.json line 125](https://github.com/EAPD-DRB/OG-ZAF/blob/main/ogzaf/ogzaf_default_parameters.json#L125) has gamma=0.61. You should change this value from 1.0 to 0.61.

cc: @SeaCelo @jdebacker